client/my-sites/domains/components/domain-warnings/pending-gapps-tos-notice-dialog.jsx

```diff
@@ -0,0 +1,158 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+import PropTypes from 'prop-types';
+import React, { Fragment, useState } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import Button from 'components/button';
+import ClipboardButton from 'components/forms/clipboard-button';
+import { composeAnalytics, recordGoogleEvent, recordTracksEvent } from 'state/analytics/actions';
+import Dialog from 'components/dialog';
+import { getLoginUrlWithTOSRedirect } from 'lib/google-apps';
+import VerticalNav from 'components/vertical-nav';
+import VerticalNavItem from 'components/vertical-nav/item';
+import wp from 'lib/wp';
+
+function PendingGappsTosNoticeDialog( props ) {
+	const [ password, setPassword ] = useState( false );
+	const [ isCopied, setIsCopied ] = useState( false );
+	const translate = useTranslate();
+
+	const onPasswordClickHandler = e => {
+		e.preventDefault;
+		const wpcom = wp.undocumented();
+		const mailbox = props.user.split( '@' )[ 0 ];
+		wpcom.resetPasswordForMailbox( props.domainName, mailbox ).then( data => {
+			setPassword( data.password );
+		} );
+	};
+
+	const recordLogInClick = () => {
+		props.pendingAccountLogInClick( {
+			domainName: props.domainName,
+			isMultipleDomains: props.isMultipleDomains,
+			user: props.user,
+			severity: props.severity,
+			section: props.section,
+			siteSlug: props.siteSlug,
+		} );
+	};
+
+	const onCopyAction = () => {
+		setIsCopied( true );
+	};
+
+	return (
+		<Dialog className="domain-warnings__dialog" isVisible={ props.visible }>
+			<header>
+				<h1>{ translate( 'Finish G Suite Setup' ) }</h1>
+				<button onClick={ props.onClose }>
+					<Gridicon icon="cross" />
+				</button>
+			</header>
+
+			<p>
+				{ password &&
+					translate(
+						'We have reset your temporary password. Please log in as `%s` with this password to finish setup:',
+						{
+							args: props.user,
+						}
+					) }
+				{ ! password &&
+					translate(
+						'In order to finish setup, you must now log into G Suite using the username and password that we sent to your email address (%s).',
+						{
+							args: props.user,
+						}
+					) }
```

Comment:
Not mandatory but we could simplify that a bit with:

```javascript
{ password ?
  translate(
    'We have reset your temporary password. Please log in as `%s` with this password to finish setup:',
    {
      args: props.user,
    }
  ) :
  translate(
    'In order to finish setup, you must now log into G Suite using the username and password that we sent to your email address (%s).',
    {
      args: props.user,
    }
  )
}
```

Reply:
I am not sure that makes this simpler.  I prefer this way as multi line ternaries hurt my brain.